canonical-web-and-design/maas-ui
# Add DNS, NTP, Syslog and Network Discovery sections to Network

## Done
- Added DNS, NTP, Syslog and Network Discovery forms to Network section

## QA
- Check that clicking on the Network subsections takes you to the correct form
- Check that saving works for each form
